data/Debian-family.yaml

```diff
@@ -40,8 +40,12 @@ galera::mariadb::104::arbitrator_package_name: 'galera-arbitrator-4'
 galera::mariadb::104::client_package_name: 'mariadb-client-10.4'
 galera::mariadb::104::galera_package_name: 'galera-4'
 galera::mariadb::104::mysql_package_name: 'mariadb-server-10.4'
+galera::mariadb::105::arbirator_package_name: 'galera-arbitrator-4'
+galera::mariadb::105::client_package_name: 'mariadb-client-10.5'
+galera::mariadb::105::galera_package_name: 'galera-4'
+galera::mariadb::105::mysql_package_name: 'mariadb-server-10.5'
 galera::mariadb::arbitrator_config_file: '/etc/default/garb'
-galera::mariadb::arbitrator_package_name: 'galera-arbitrator-3'
+galera::mariadb::arbitrator_package_name: 'galera-arbitrator-4'
```

Comment:
This should probably not be altered at this point. Would you add this setting to `data/Debian-11.yaml`?

Reply:
Hi, I added Debian-11.yaml and revert my changed in the family file.